freqtrade/strategy/interface.py

```diff
@@ -467,6 +467,28 @@ def adjust_trade_position(self, trade: Trade, current_time: datetime,
         """
         return None
 
+    def readjust_entry_price(self, trade: Trade, pair: str, current_time: datetime,
+                             proposed_rate: float, entry_tag: Optional[str],
+                             side: str, **kwargs) -> float:
```

Comment:
i somehow don't like the naming - but lack a better name for now.

it somehow looks too much like "read just" ... which won't make any sense.
maybe splitting it (re_adjust_entry_price) makes it better.

That said - i would think that what this method is actually doing is not readjusting the price - but readjusting the order - therefore something like  `adjust_open_order()` might also make sense?

Reply:
Initially it was `adjust_entry_price` and I wanted to add emphasis on the `re`occuring nature. `adjust_open_order` is fine for me 😛

Reply:
Considering we're taking `entry` out of the name, should this handle both entries and exits?